src/renderer/components/modules/AddAccountWindow.tsx

```diff
@@ -57,6 +60,21 @@ const AddAccountWindow = (props: AddAccountWindowProps) => {
   const [loading, setLoading] = useState(false);
   const [errorMsg, setErrorMsg] = useState("");
 
+  const [openai_api_key, setApiKey] = useState("");
```

Comment:
JavaScript uses camelCase convention for variable naming. Consider `apiKey` or `openaiApiKey`.